packages/backend-defaults/src/entrypoints/auth/user/UserTokenHandler.ts

```diff
@@ -134,6 +134,12 @@ export class UserTokenHandler {
       );
     }
 
+    // If payload.uip is missing, return the original token as we cannot create
+    // a valid limited token without the user identity proof signature
+    if (!payload.uip) {
+      return { token: backstageToken, expiresAt: new Date(payload.exp * 1000) };
```

Comment:
This isn't right.

It's important that what you get back actually DOES have limited capabilities. We explicitly call this method to get a token that isn't wielding the full power of the user; if that contract were broken, we would be exposing systems to new risk.